src/Types.hs

```diff
@@ -68,16 +93,42 @@ data IdeaSpacePhase =
 
 -- | "Idee"
 data Idea = Idea
-    { _ideaMeta       :: MetaInfo
+    { _ideaMeta       :: MetaInfo Idea
     , _ideaTitle      :: ST
-    , _ideaArticle    :: Article
+    , _ideaDesc       :: Document
     , _ideaCategory   :: Category
+    , _ideaPhase      :: Phase
+    , _ideaTopic      :: Maybe Topic
     , _ideaComments   :: Set Comment
-    , _ideaVotes      :: Set Vote
-    , _ideaInfeasible :: Maybe ST  -- ^ Reason for infisibility, if any.
+    , _ideaLikes      :: Set Like
+    , _ideaVotes      :: Set IdeaVote
+    , _ideaFeasible   :: Maybe Feasible
     }
   deriving (Eq, Ord, Show, Read, Generic)
 
+data Feasible = Feasible
+    { _feasibleMeta   :: MetaInfo Feasible
+    , _feasibleValue  :: Bool
+    , _feasibleReason :: Document
+    }
+  deriving (Eq, Ord, Show, Read, Generic)
+
+-- | FIXME: Is there a better name for 'Like'?  'Star'?  'Endorsement'?  'Interest'?
+data Like = Like
+    { _likeMeta  :: MetaInfo Like
+    }
+  deriving (Eq, Ord, Show, Read, Generic)
```

Comment:
I'm slightly inclined to call this `IdeaLike` to mirror the use of `IdeaVote` ...